package.json

```diff
@@ -139,7 +139,7 @@
     "react-native-google-places-autocomplete": "2.5.6",
     "react-native-haptic-feedback": "^2.2.0",
     "react-native-image-picker": "^7.0.3",
-    "react-native-image-size": "git+https://github.com/Expensify/react-native-image-size#8393b7e58df6ff65fd41f60aee8ece8822c91e2b",
+    "react-native-image-size": "file:../react-native-image-size/react-native-image-size-1.1.3.tgz",
```

Comment:
Don't forget to update this once https://github.com/Expensify/react-native-image-size/pull/4#pullrequestreview-1952288909 gets merged